package.json

```diff
@@ -21,5 +23,6 @@
     "alex": "^10.0.0",
     "eslint": "^8.10.0",
     "prettier": "^2.5.1"
-  }
+  },
+  "prettier": "@readme/eslint-config/prettier"
```

Comment:
Might want to add a `private: true` to this file so it doesn't ever get somehow accidentally published to NPM.